portlist/portlist_windows.go

```diff
@@ -33,7 +33,7 @@ type portMeta struct {
 	keep bool
 }
 
-func newWindowsImpl() osImpl {
+func newWindowsImpl(includeLocalhost bool) osImpl {
 	return &windowsImpl{
 		known: map[famPort]*portMeta{},
 	}
```

Comment:
not doing windows? and openbsd/freebsd/etc?

kinda a bummer that any non-portable code needs to know about this.

seems like it'd be nice if the OS-specific files just included all ports (even localhost) internally and then the portable layer did the localhost filtering before the process name lookup

Reply:
@bradfitz I just missed it between rebases, pushing an update in the next commit.